test/test.mk

```diff
@@ -1,7 +1,7 @@
 NAMESPACE ?= userdb
 POSTGRES_USER ?= postgres
 POSTGRES_DB ?= postgres
-IMAGE_POSTGRES ?= postgres:latest
+IMAGE_POSTGRES ?= postgres@sha256:3657548977d593c9ab6d70d1ffc43ceb3b5164ae07ac0f542d2ea139664eb6b3
```

Comment:
Generally the base version of postgres changing isn't going to affect postgrests ability to function or be backwards compatible

Reply:
Turns out it did. Latest postgres actually breaks postgrest in a bunch of weird ways. Tests were failing until I backp-pinned postgres. There are a few issues on it, but a problem for another day.

Reply:
Probably because we're not using latest postgrest 😉